_drafts/2021-03-15-data_science_timeseries_plotting_notebook_template.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: "Jupyter Notebook Templates for Data Science: Plotting Time Series"
+description: >
+  Jumpstart your time series visualizations with this Jupyter plotting notebook!
+image: /files/jupyter-library/jupiter_in_the_rearview_mirror.jpg
+image_alt: >
+  The planet Jupiter as seen by the departing Juno spacecraft.
+categories: 
+  - data-visualization
+  - data-science
+  - jupyter
+  - my-projects
+---
+
+{% capture file_dir %}/files/jupyter-library/{% endcapture %}
+
+I often have data where each row describes an event that happened at some
+time. The data might describe [a word that my son spoke for the first
```

Comment:
```suggestion
I often have data where each row describes an event. 
The data might describe [a word that my son spoke for the first
```
Wording is clunky here. I think some of this work belongs in a different sentence